crates/prune/src/builder.rs

```diff
@@ -102,7 +102,7 @@ impl Default for PrunerBuilder {
             segments: PruneModes::none(),
             max_reorg_depth: 64,
             prune_delete_limit: MAINNET.prune_delete_limit,
-            timeout: Some(Self::DEFAULT_TIMEOUT),
+            timeout: None,
```

Comment:
let's remove the constant all together then?

Reply:
it's still used in the node builder https://github.com/paradigmxyz/reth/blob/a8bbab2470c08140f10d327989a1ac3122ca2211/crates/node/builder/src/launch/common.rs#L281